src/vs/platform/files/common/inMemoryFilesystemProvider.ts

```diff
@@ -11,7 +11,7 @@ import { URI } from 'vs/base/common/uri';
 
 class File implements IStat {
 
-	type: FileType;
+	type: FileType.File;
```

Comment:
Changed it to be able to handle types inside tests in a nicer way. Since it only makes typings here stricter, I've though it's a good change.